salt/modules/boto3_route53.py

```diff
@@ -1000,3 +1003,448 @@ def change_resource_record_sets(HostedZoneId=None, Name=None,
+    return data['HealthCheckVersion'] if data else None
+
+def create_health_check(Name, CallerReference=None, IPAddress=None,
+                        Port=None, Type=None, ResourcePath=None, FullyQualifiedDomainName=None, SearchString=None,
+                        RequestInterval=None, FailureThreshold=None, MeasureLatency=None, Inverted=None, Disabled=None,
+                        HealthThreshold=None, ChildHealthChecks=None, EnableSNI=None, Regions=None, AlarmIdentifier=None,
+                        InsufficientDataHealthStatus=None, region=None, key=None, keyid=None, profile=None, tries=20):
+    '''
+    Creates a new health check with given properties.
+
+    Name
+        The name to assign to newly created health check.
+    region
+        Region to connect to.
+    key
+        Secret key to be used.
+    keyid
+        Access key to be used.
+    profile
+        Dict, or pillar key pointing to a dict, containing AWS region/key/keyid.
+    CLI Example:
+    .. code-block:: bash
+        salt myminion boto3_route53.create_health_check ANAME ... \
+                profile='{"region": "us-east-1", "keyid": "A12345678AB", "key": "xblahblahblah"}'
+    '''
+    if not Name:
+        raise SaltInvocationError('Name of healthcheck to update missing.')
+
+    keys = [
+        'IPAddress', 'Port', 'Type', 'ResourcePath', 'FullyQualifiedDomainName', 'SearchString',
+        'RequestInterval', 'FailureThreshold', 'MeasureLatency', 'Inverted', 'Disabled', 'HealthThreshold',
+        'EnableSNI', 'InsufficientDataHealthStatus', 'Regions', 'AlarmIdentifier', 'ChildHealthChecks'
+    ]
+    params = locals()
+    result = None
+
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+    CallerReference = CallerReference if CallerReference else str(uuid.uuid4())  # future lint: disable=blacklisted-function
+    HealthCheckConfig = {}
+
+    for key in [x for x in keys if params.get(x) is not None]:
+        HealthCheckConfig[key] = params.get(key)
+
+    args = {'CallerReference': CallerReference, 'HealthCheckConfig': HealthCheckConfig}
+
+    while tries:
+        try:
+            result = conn.create_health_check(**args).get('HealthCheck')
+            conn.change_tags_for_resource(
+                ResourceType='healthcheck',
+                ResourceId=result['Id'],
+                AddTags=[{ 'Key': 'Name', 'Value': Name }],
+                )
+            return {'result': result }
+        except ClientError as e:
+            if tries and e.response.get('Error', {}).get('Code') == 'Throttling':
+                log.debug('Throttled by AWS API.')
+                time.sleep(3)
+                tries -= 1
+                continue
+            log.error('Failed to create healthcheck %s: %s', Name, six.text_type(e))
+            raise e
+
+    return result
+
+def update_health_check(Id, changes, region=None, key=None, keyid=None, profile=None, tries=20):
```

Comment:
I think we should prefer `id` here, too

Reply:
This is another case of code based on a pattern already existing on this same module (ie. find_hosted_zone)